spec/lib/entity_cache_spec.rb

```diff
@@ -16,4 +16,11 @@
       end
     end
   end
+  describe '#to_emoji' do
+    context 'when input shortcode has cases'
+      it 'returns emoji with cases preserved' do
+        expect(described_class.instance.to_emoji(:emoji, 'foobar', 'example.org')).not_to eq described_class.instance.to_emoji(:emoji, 'FooBar', 'example.org')
```

Comment:
I'd probably frame this as `expect(method).to eq "fixed string"`. In this case something like:

`expect(described.instance.to_emoji(...)). to eq "emoji:foobar:example.org"` or similar.

Reply:
Maybe I could write something like expect(described_class.instance.to_emoji(:emoji, 'FooBar', 'example.org').to eq "emoji:FooBar:example.org"? This should also do the trick. I'll update this soon when I get back to work :-)